README.md

```diff
@@ -9,106 +9,236 @@
 - Document how a component behaves in various scenarios.
 - Get a11y, snapshot and visual tests using other addons for free.
 
+[Full documentation and live demos](https://msw-sb.vercel.app/)
+
+## Installing and Setup
+
+### Install MSW and the addon
+
+```sh
+npm i -D msw msw-storybook-addon
```

Comment:
A minor comment but I generally recommend putting the flags _after_ the installed package names. That way the most important things come first:

- what command? `npm`
- what subcommand? `install`
- install what? `msw msw-storybook-addon`
- install how? `-D`

Reply:
good one!